common/src/main/java/com/wynntils/utils/KeyboardUtils.java

```diff
@@ -13,10 +13,14 @@ public static boolean isKeyDown(int keyCode) {
     }
 
     public static boolean isShiftDown() {
-        return isKeyDown(GLFW.GLFW_KEY_RIGHT_SHIFT) || isKeyDown(GLFW.GLFW_KEY_LEFT_SHIFT);
+        return isKeyDown(GLFW.GLFW_KEY_LEFT_SHIFT) || isKeyDown(GLFW.GLFW_KEY_RIGHT_ALT);
```

Comment:
Doesn't this have to be GLFW_KEY_RIGHT_SHIFT like it used to be instead of GLFW.GLFW_KEY_RIGHT_ALT